patches/octane_run_we.js

```diff
@@ -0,0 +1,51 @@
+load('base.js');
+load('richards.js');
+load('deltablue.js');
+load('crypto.js');
+load('raytrace.js');
+load('earley-boyer.js');
+load('regexp.js');
+load('splay.js');
+load('navier-stokes.js');
+load('pdfjs.js');
+//load('mandreel.js');
+//load('gbemu-part1.js');
+//load('gbemu-part2.js');
+//load('code-load.js');
+//load('box2d.js');
+//load('zlib.js');
+//load('zlib-data.js');
+load('typescript.js');
+load('typescript-input.js');
+load('typescript-compiler.js');
+
+
+function main() {
+  var suites = BenchmarkSuite.suites;
+  for (var i = 0; i < suites.length; i++) {
+    var suite = suites[i];
+    var benchmarks = suite.benchmarks;
+    for (var j = 0; j < benchmarks.length; j++) {
```

Comment:
The first two loops just deal with Octane's particular data-structure (benchmark suites have benchmarks). There's nothing very deep in this.